ansible/roles/jupyterhub/files/jupyterhub_config_lti11.py

```diff
@@ -87,8 +87,8 @@
 c.JupyterHub.authenticator_class = LTI11Authenticator
 
 # Spawn containers with by role or workspace type
-c.JupyterHub.spawner_class = IllumiDeskRoleDockerSpawner
-# c.JupyterHub.spawner_class = IllumiDeskWorkSpaceDockerSpawner
+# c.JupyterHub.spawner_class = IllumiDeskRoleDockerSpawner
```

Comment:
We should probably set the `IllumiDeskRoleDockerSpawner` as the default for both LTI 1.1 and LTI 1.3 until we update the configuration options with #148